docs/sources/alerting/notifications.md

```diff
@@ -111,6 +111,19 @@ Token | If provided, Grafana will upload the generated image via Slack's file.up
 
 If you are using the token for a slack bot, then you have to invite the bot to the channel you want to send notifications and add the channel to the recipient field.
 
+### Opsgenie
+
+To setup Opsgenie you will need an API Key and the Alert API Url. These can be obtained by configuring a new [Grafana Integration](https://docs.opsgenie.com/docs/grafana-integration).
+
+Setting | Description
+--------|------------
+Alert API URL | The API URL for your Opsgenie instance. This will normally be either `https://api.opsgenie.com` or, for EU customers, `https://api.eu.opsgenie.com`.
+API Key | The API Key as provided by Opsgenie for your configured Grafana integration.
+Override priority | Configures the alert priority using the `og_priority` tag. The `og_priority` tag must have one of the following values: `P1`, `P2`, `P3`, `P4`, or `P5`. Default is `False`.
+Send notification tags as | Specify how you would like [Notification Tags]({{< relref "create-alerts.md/#notifications" >}}) delivered to Opsgenie. They can be delivered as `Tags`, `Extra Properties` or both. Default is `Tags`. (1)
```

Comment:
What is `(1)` referring to at the end here? This is the only thing left as I see to get this merged.

Reply:
@marefr It's the first note below. I took inspiration from the PagerDuty docs that used the same mechanism to refer to notes below. Maybe it would be a bit clearer if I added the `(1)` to the Note itself to link it back and made a link. Give me a few minutes to get that done.

Reply:
Hmm okay. Not sure. Maybe keeping the note and refer to "Default is `Tags`. See note below for more information.". Lets ask @achatterjee-grafana for input on what make most sense.

Reply:
Agree with @marefr. This is not consistent with our style guide and use. It would be confusing to the reader, as evident from @marefr's reaction. Let's change it as per the recommendation!

Reply:
Can do I'll push another fix.